### PR TITLE
[integ-tests] Update develop.yaml and isolated_regions.yaml

### PR DESCRIPTION
**Pull request by hanwen-pcluste**
There were missing tests in the yaml files.

Moreover, this commit moves the location of test_cluster_with_gpu_health_checks in isolated_regions so that it is in the same location as develop.yaml. This was done to help me compare the two files.

### References
* Link to impacted open issues.
* Link to related PRs in other packages (i.e. cookbook, node).
* Link to documentation useful to understand the changes.

### Checklist
- Make sure you are pointing to **the right branch**.
- If you're creating a patch for a branch other than `develop` add the branch name as prefix in the PR title (e\.g\. `[release-3.6]`).
- Check all commits' messages are clear, describing what and why vs how.
- Make sure **to have added unit tests or integration tests** to cover the new/modified code.
- Check if documentation is impacted by this change.

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integration-tests/configs/isolated_regions.yaml [hunk 1]
```diff
@@ -102,31 +102,38 @@ test-suites:
 #          instances: {{ INSTANCES }}
 #          oss: {{ OSS }}
 #          schedulers: {{ SCHEDULERS }}
-    test_create.py::test_create_wrong_pcluster_version:
+# test_create_wrong_pcluster_version is a test on a corner case. It is not compatible with isolated regions
```
Comment (himani2411) [inline tests/integration-tests/configs/isolated_regions.yaml]: why is it not compatible for Isolated regions?

**Review by himani2411 — APPROVED** [review]
